src/main/java/it/gov/pagopa/payhub/auth/service/user/IamUserInfoDTO2UserInfoMapper.java

```diff
@@ -115,4 +117,21 @@ private Broker getSessionBroker(IamUserInfoDTO iamUserInfoDTO, List<Operator> us
         return null;
     }
 
+    private void setBrokerInfo(UserInfo userInfo, String accessToken) {
+        IamUserInfoDTO iamUserInfo = tokenStoreService.load(accessToken);
+        if (iamUserInfo == null) {
+            throw new InvalidAccessTokenException("AccessToken not found");
+        }
```

Comment:
why are you doing this? invocation of the method you already have IamUserInfoDTO